setup.py

```diff
@@ -42,7 +42,16 @@
 # This is used to deal with the fact that we have multiple packages which
 # might be built optionally all tracked / installed via one setup.
 exclude_packages = [x.strip() for x in os.environ.get('EXCLUDE_PACKAGES', '').split(',') if x != '']
-exclude_packages.extend(['subscription_manager.gui.firstboot.*', '*.ga_impls', '*.ga_impls.*', '*.plugin.ostree', '*.services.examples', 'syspurpose*'])
+exclude_packages.extend(
+    [
+        'subscription_manager.gui.firstboot.*',
+        '*.ga_impls',
+        '*.ga_impls.*',
+        '*.plugin.ostree',
+        '*.services.examples',
+        'syspurpose*'
```

Comment:
```suggestion
        'syspurpose*',
        'subscription_manager.migrate',
```

Reply:
This will need to be added and removed from this list just like the others based on whether or not we are building migration-data.

Reply:
Adding `subscription_manager.migrate` to this list here would exclude building the `subscription_manager.migrate` in all situations (e.g. rhel7 and rhel8). We don't want such behavior.